packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -510,6 +509,7 @@ export class VaTextInput {
               part="input"
               min={min}
               max={max}
+              value={value}
```

Comment:
Moving this attribute to below min/max resolves this Stencil test warning:

![Screenshot 2024-11-18 at 2 01 49 PM](https://github.com/user-attachments/assets/e32cff48-c104-483b-a537-cb4780c73fa3)